srcbook/bin/start

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env node
 
-import '../index.mjs';
+process.env.NODE_ENV = 'production';
```

Comment:
Can we use dotenv here, or some other more standard way? We're going to eventually want to set more flags in prod anyways.

Reply:
Mind if I do this in a follow up? This PR is already blown up a bit.